### ISSUE TITLE
The byte-order mark (BOM) in HTML   14 Apr 2023

### ISSUE THREAD

**Issue by r12a** [w3c/i18n-translations#57]
https://w3c.github.io/i18n-drafts/questions/qa-byte-order-mark.en.html
https://w3c.github.io/i18n-drafts/questions/qa-byte-order-mark.fr.html

Translator: Gwen Clavé, Clavoline Traduction (@clavoline)
https://lists.w3.org/Archives/Public/public-i18n-translation/2023AprJun/0000.html

**Comment by clavoline** [comment]
I think there is an issue with the `Quick check` in the right-side column:

![image](https://user-images.githubusercontent.com/120583347/232598683-06302a2d-1dec-4cdb-baaf-9a8aeb010725.png)

When copy-pasting a URI into the input field and clicking `Check`, I expect to see the results of the check. For instance, if I enter `https://www.w3.org/International/questions/qa-byte-order-mark`, I expect to see this:

![image](https://user-images.githubusercontent.com/120583347/232600547-0eda9feb-72aa-4993-b440-e0e899b27bc3.png)

Here is what I see instead:

![image](https://user-images.githubusercontent.com/120583347/232600645-d7ffc714-11fb-4a5d-81f9-796dbccbb0f5.png)

As a result, I need to paste my URI again on that page to get my results.

In other words, the button takes me to `https://validator.w3.org/i18n-checker/?docAddr=https%3A%2F%2Fwww.w3.org%2FInternational%2Fquestions%2Fqa-byte-order-mark`, but the page is the same as `https://validator.w3.org/i18n-checker/` -- adding the parameter doesn't change anything.

Not a translation issue, but I thought I should point this out.

**Comment by r12a** [comment]
Thanks for the report, Gwen.  I have sent a request to our systems folks to look into why this no longer works.

**Comment by r12a** [comment]
Ok, we have a solution.

`i18n-checker/?docAddr=` needs to be replaced with `i18n-checker/check?uri=`

Thanks again for spotting this.

**Comment by clavoline** [comment]
That was fast!

Thank you, I'll make the change in both versions (unless it's already been done in the English version).

**Comment by r12a** [comment]
Updates complete.  I just updated the de version too.
